readthedocs/projects/version_handling.py

```diff
@@ -108,7 +108,7 @@ def version_windows(versions, major=1, minor=1, point=1):
 def parse_version_failsafe(version_string):
     try:
         return Version(version_string)
-    except InvalidVersion:
+    except (InvalidVersion, UnicodeEncodeError):
```

Comment:
Oops! Sorry, this change isn't meant to be here. It's a workaround for https://github.com/pypa/packaging/issues/90 which I had in place but didn't intend to commit. I'll remove it.